system-tests/_scripts/sandbox-setup.sh

```diff
@@ -4,4 +4,6 @@ SCRIPT=`realpath $0`
 SCRIPT_PATH=`dirname $SCRIPT`
 CYPRESS=`realpath "${SCRIPT_PATH}/../../node_modules/cypress/bin/cypress"`
 
-ln -s $CYPRESS "$TMPDIR/cypress"
\ No newline at end of file
+cp -R "/root/.cache" "$TMPDIR/../"
```

Comment:
❔ What manages `/root/.cache`? Jenkins? Does this script work in local dev environment?

Reply:
The script is supposed to be ran in docker even in local dev environment. So the user is `root`.

Reply:
I would normally say let's just make it configurable, but the sandbox setup within system tests is pretty specific to being run on docker